bundles/core/src/main/java/com/adobe/cq/commerce/demandware/preview/PreviewServiceImpl.java

```diff
@@ -260,7 +261,7 @@ protected CredentialsProvider createCredentialsProvider(Map<String, Object> conf
         if (PropertiesUtil.toString(configuration.get(STORFRONT_PROTECTION_USER), null) != null && PropertiesUtil
                 .toString(configuration.get(STORFRONT_PROTECTION_PASSWORD), null) != null) {
             final CredentialsProvider credsProvider = new BasicCredentialsProvider();
-            credsProvider.setCredentials(new AuthScope(demandwareClient.getEndpoint(), AuthScope.ANY_PORT),
+            credsProvider.setCredentials(new AuthScope(clientProvider.getDefaultClient().getEndpoint(), AuthScope.ANY_PORT),
```

Comment:
we have probably two instances here.. and maybe not even a default one. this would mean, we cannot preview on a multi-instance setup.